Competition/src/test/java/competition/subsystems/elevator/ElevatorSubsystemTest.java

```diff
@@ -83,7 +78,32 @@ public void testGetCalibrationHeight() {
         elevatorSubsystem.calibrate();
         assertEquals(0, elevatorSubsystem.getCalibrationHeight(), 0.001);
     }
-
+    @Test
+    public void testSetPower() {
+        assertEquals(0.0, elevatorSubsystem.getMasterPower(), 0.001);
+        assertTrue(contract.isElevatorReady());
+        assertTrue(contract.isElevatorLimitSwitchReady());
+        testIsCalibrationSensorPressed();
+        assertTrue(elevatorSubsystem.activateBrakeSolenoid.getAdjusted());
+
+        elevatorSubsystem.setPower(1);
```

Comment:
It might be easier to read these test cases if they were broken out into 3 separate test methods with descriptive names, eg: testSetPowerZero etc..